src/main/java/seedu/address/model/person/Priority.java

```diff
@@ -0,0 +1,8 @@
+package seedu.address.model.person;
+
+/**
+ * Available priority levels for tasks and events.
+ */
+public enum Priority {
```

Comment:
If I am not wrong, there is a need to implement comparable so that we are able to compare across the enumerations.